bin/serve

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+
+const Express = require('express');
+const ExpressHandlebars = require('express-handlebars');
+const BodyParser = require('body-parser');
+const path = require('path');
+const http = require('http');
+const apiApp = require('../api/app');
+
+const apiService = require('../src/services/api');
+const config = require('../src/config.json');
+
+
+// create static app
+const urlencoded = BodyParser.urlencoded({
+  extended: false,
+});
+const staticApp = Express();
+const staticRootDir = path.join(__dirname, '..', 'build');
+
+staticApp.use(urlencoded);
+staticApp.use(Express.static(staticRootDir, {index: false}));
+
+staticApp.set('views', staticRootDir);
+staticApp.engine('html', ExpressHandlebars({extname: '.html', layoutsDir: staticRootDir}));
+staticApp.set('view engine', 'html');
+
+staticApp.get('/plan/:planId/:planName?', function(req, res) {
+  const { planId, planName } = req.params;
+
+  // fetch plan details to populate og tags
+  apiService.get('/plan/' + planId)
+    .then(plan => {
+      let pageDescription;
+
+      // some plans don't have depositing dates
+      if (plan.data.data.DEPOSITING_DATE)
+        pageDescription = `תב"ע ב${plan.data.data.PLAN_COUNTY_NAME} אשר הופקדה ב ${dateToStr(plan.data.data.DEPOSITING_DATE)}`;
+      else
+        pageDescription = `תב"ע ב${plan.data.data.PLAN_COUNTY_NAME}`;
+
+      res.render('index', {
+        layout: false, 
+        pageTitle: `${plan.data.PL_NUMBER} - ${plan.data.PL_NAME} - מעירים`, 
+        pageDescription: pageDescription,
+        pageImage: config.opengraph.basePageImageURL,
+      });
+    }).catch(error => {
+      console.error('failed to fetch plan details for opengraph');
+      console.error(error);
+
+      res.render('index', {
+        layout: false, 
+        pageTitle: 'תוכנית בניה - מעירים', 
+        pageDescription: '',
+        pageImage: config.opengraph.basePageImageURL,
+      });
+    });
+});
+
+// all non-plan pages have static og tag values
+staticApp.get('*', function(req, res) {
```

Comment:
This doesn't make sense since we should return errors for unexisting urls.

Reply:
same as above - we shouldn't return errors for the static app so the react app loads and shows the react 404 page